src/controllers/tripsController.js

```diff
@@ -502,13 +473,38 @@ class tripsController {
         }
       );
       const name = `${firstName} ${lastName}`,
-      notification = ` ${name} edited a trip request.`;
+        notification = ` ${name} edited a trip request.`;
       await sendEditNotification(req.user, 'edited trip request', tripId, notification, host);
       return successResponse(res, 200, setLanguage(preferedLang).__('TriprequestUpdatedSuccess'), fineTripRequest);
     } catch (error) {
       return errorResponse(res, 500, error.message);
     }
   }
+
+  /**
+   * @description Find most traveled destinations
+   * @param {object} req
+   * @param {object} res
+   * @memberof tripsController
+   * @returns {object} response
+   */
+  static async getMostTravelledCentres(req, res) {
+    try {
+      const {
+        user: {
+          preferedLang
+        },
+        query: {
+          page, limit,
+        }
+      } = req;
+      const mostTraveledCentres = await tripService.getTrendingCentres(page, limit);
+
+      return successResponse(res, 200, setLanguage(preferedLang).__('trendingDestinations'), mostTraveledCentres);
+    } catch (error) {
+      return errorResponse(res, 500, error.message);
+    }
+  }
```

Comment:
may you please put it in the accommodation controller

Reply:
@blaise82 Thanks for FRESH EYES. I fixed the issue